policy/modules/contrib/insights_core.te

```diff
@@ -193,6 +194,7 @@ optional_policy(`
 optional_policy(`
 	gnome_search_gconf(insights_core_t)
 	gnome_manage_generic_cache_files(insights_core_t)
+	gnome_filetrans_cert_home_content(insights_core_t)
```

Comment:
This also requires some justification, services usually should not need to access user data.

Reply:
It's a pity the full path is not audited. If it is /var/lib/flatpak/repo/objects, the directory is mislabeled. If it is in the user's home directory, it should not access it.

Reply:
Reproduced, seems to be the way how flatpak works. Still does not feel right.